airflow/dag_processing/processor.py

```diff
@@ -647,3 +648,17 @@ def process_file(
             self.log.exception("Error logging import errors!")
 
         return len(dagbag.dags), len(dagbag.import_errors)
+
+    def delete_serialized_dags_with_parsing_errors(self, session, file_path):
+        from airflow.models.serialized_dag import SerializedDagModel
+
+        self.log.error("Deleting Serialized DAG(s) in %s file, please fix the parsing errors.", file_path)
```

Comment:
If a DAG file has a parsing error, this will be logged every single time the file is processed, even though the serialized DAG is only deleted the first time that this function is run.

Maybe we should only log when a serialized run is actually deleted?

Reply:
Good point, yes